Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,68 @@
+﻿/*
+ * ********Lesson Notes for me here********
+ * Requirements for this bootcamp lesson(lesson 3)
+ *  1. Need to have refactored code to recieve inputs
+ *  2. Need to gather the following pieces of information:
+ *      -Name(string}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4862917)